src/components/Site/Logger.tsx

```diff
@@ -32,21 +36,58 @@ export function LoggerTab({
     logger,
     logger_uid,
 }: LoggerTabProps): ReactElement {
+    const channelTemplates = useSelector((state: RootState) => state.templates);
+
     const [infoExpanded, setInfoExpanded] = useState(false);
 
     const handleInfoButton = () => setInfoExpanded(!infoExpanded);
 
-    const columns: GridColDef[] = [
-        { field: 'timestamp', headerName: 'timestamp', flex: 1 },
+    const dateFormat = 'M/D/YYYY hh:mm:ss:SSS A';
+
+    const [
+        gridRef,
+        setGridRef,
+    ] = useState<MutableRefObject<TypeComputedProps | null> | null>(null);
+
+    const columns: any[] = [
+        {
+            name: 'timestamp',
+            header: 'Timestamp',
+            minWidth: 150,
+            defaultFlex: 1,
+            dateFormat: dateFormat,
+            filterEditor: DateFilter,
+            filterEditorProps: () => {
+                return { dateFormat: dateFormat };
+            },
+        },
     ];
 
+    const filters = [
+        { name: 'timestamp', operator: 'after', type: 'date', value: '' },
+    ];
+
+    for (const [id, data] of Object.entries(channelTemplates)) {
```

Comment:
Just get the template directly instead of iteration over all of them.

Reply:
I already fixed this one.